docs/development/release.md

```diff
@@ -110,6 +126,15 @@ The CI release scripts will automatically create a release based on the release
 
 Just make sure to verify the appropriate .zip files are present on Github after the release process is complete.
 
+### 10. Merge base branch back to master
+
+Finally, you need to reproduce the version update changes back to master.
+
+If this is the latest version, you can simply create a PR to merge the minor version base branch back to master.
+
+If this is a backport, you will need to create a PR that ports versioned docs from step 5 and the changelog updates back to master.
+You will need to ensure that the backport's versioned docs do not end up as the default [published documentation](https://nasa.github.io/cumulus), but that the default remains the most highly-numbered version released (e.g. we do not want 1.14.5 to show up by default instead of 1.17.0.)
```

Comment:
Have we figured out a good approach for this? I could edit to specify that the CHANGELOG should be merged back but versioned docs are ideal but optional.

Reply:
No.   This has some significant issues.    Firstly, as we're publishing master all of the time, any previous version doc updates need to be in master to be published.    But to PR the changes back to master, you have to pull in only doc changes while retaining master in the PR to master. 

My vote is we *avoid* making doc updates in prior non-master-version backports.   We shouldn't be making non-maintenance changes in backports, and if we *do* we should make a PR to master and backport the change *from* master if we're updating a prior version.

I say that being aware that that's probably not how docusarus works.     We should open a ticket into sorting this if we don't have one already. 

Changelogs were agreed to be ported forward, with an explicit notice that the change is  a backport and only applies to the current minor (or whatever it is outside semver) version.